e2e/tests/nightly/annual-access-renewal.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { signInWithAccessToken } from 'utils/test-utils';
+import { config } from 'resources/workbench-config';
+import AccessRenewalPage from 'app/page/access-renewal-page';
+import ProfilePage from 'app/page/profile-page';
+import ProfileConfirmationModal from 'app/modal/profile-confirmation-modal';
+import Navigation, { NavLink } from 'app/component/navigation';
+import HomePage from 'app/page/home-page';
+
+// Important: the access test user must be in a state where they are currently failing access renewal
+// due to an expired "profile last confirmed" date.  CircleCI accomplishes this by running the access-test-user-setup
+// task, which executes `./project.rb set-access-module-timestamps --user ${PUPPETEER_ACCESS_TEST}`.
+// This can also be run locally.
+
+describe('Annual Access Renewal', () => {
+  beforeEach(async () => {
+    await signInWithAccessToken(page, config.ACCESS_TEST_ACCESS_TOKEN_FILE);
+  });
+
+  // This test cannot run after the "Expired User can complete Annual Access Renewal" test - see below.
+
+  test('Expired User is redirected to Annual Access Renewal (AAR) on login', async () => {
+    const aarPage = await new AccessRenewalPage(page).waitForLoad();
+    expect(aarPage).toBeTruthy();
+    expect(await aarPage.hasExpired()).toBeTruthy();
+  });
+
+  // note that this test is "destructive" in that it brings the user to a state
+  // where they cannot complete this test again, because they have completed
+  // AAR and are no longer forced into renewal
+
+  test('Expired User can complete Annual Access Renewal (AAR)', async () => {
```

Comment:
`test('Expired User is redirected to Annual Access Renewal (AAR) on login'` and `test('Expired User can complete Annual Access Renewal (AAR)'` should be one test to avoid extra page close/open/login.